plugins/inputs/ceph/ceph.go

```diff
@@ -345,6 +345,8 @@ type CephStatus struct {
 		ReadBytesSec  float64 `json:"read_bytes_sec"`
 		WriteBytesSec float64 `json:"write_bytes_sec"`
 		OpPerSec      float64 `json:"op_per_sec"`
+		ReadOpPerSec  float64 `json:"read_op_per_sec"`
+		WriteOpPerSec float64 `json:"write_op_per_sec"`
```

Comment:
Let's make the three fields a `*float64`, if the value is nil it will be skipped over when adding it to the accumulator.  Can you also add a comment with the version of ceph that stopped exporting the field here as well.

Reply:
One issue with making these 3 fields pointers is that if there isn't any client I/O going on ceph doesn't report the fields. This means we can't differentiate between zero I/O and no exported data, but we do always want to report client I/O

Reply:
Okay, that being the case I agree we should leave it as is.